manifests/profile/rabbitmq.pp

```diff
@@ -47,6 +50,42 @@
       before   => Class['rabbitmq::repo::rhel'],
     }
   }
+  elsif ($facts['os']['family'] == 'Debian') {
```

Comment:
Should we get rid of the RHEL version check above this so that everything is using the same repo for erlang?